docs/template_plugin/tests/functional/shared_tests_instances/behavior/ov_executable_network/properties.cpp

```diff
@@ -0,0 +1,120 @@
+                        OVCompiledModelPropertiesIncorrectTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Hetero_BehaviorTests, OVCompiledModelPropertiesIncorrectTests,
+                        ::testing::Combine(
+                                ::testing::Values(CommonTestUtils::DEVICE_HETERO),
+                                ::testing::ValuesIn(hetero_inproperties)),
+                        OVCompiledModelPropertiesIncorrectTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_Multi_BehaviorTests, OVCompiledModelPropertiesIncorrectTests,
+                        ::testing::Combine(
+                                ::testing::Values(CommonTestUtils::DEVICE_MULTI),
+                                ::testing::ValuesIn(multi_inproperties)),
+                        OVCompiledModelPropertiesIncorrectTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_Auto_BehaviorTests, OVCompiledModelPropertiesIncorrectTests,
+                        ::testing::Combine(
+                                ::testing::Values(CommonTestUtils::DEVICE_AUTO),
+                                ::testing::ValuesIn(auto_inproperties)),
+                        OVCompiledModelPropertiesIncorrectTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(DISABLED_smoke_AutoBatch_BehaviorTests, OVCompiledModelPropertiesIncorrectTests,
+                        ::testing::Combine(
+                                ::testing::Values(CommonTestUtils::DEVICE_BATCH),
+                                ::testing::ValuesIn(auto_batch_inproperties)),
+                        OVCompiledModelPropertiesIncorrectTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> default_properties = {
+        {ov::enable_profiling(true)},
+        {ov::device::id("0")},
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, OVCompiledModelPropertiesDefaultTests,
+        ::testing::Combine(
+                ::testing::Values(CommonTestUtils::DEVICE_TEMPLATE),
+                ::testing::ValuesIn(default_properties)),
+        OVCompiledModelPropertiesDefaultTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> properties = {
+        {ov::enable_profiling(true)},
+        {ov::device::id("0")},
+};
+
+const std::vector<ov::AnyMap> hetero_properties = {
+        {ov::device::priorities(CommonTestUtils::DEVICE_TEMPLATE), ov::enable_profiling(true)},
+        {ov::device::priorities(CommonTestUtils::DEVICE_TEMPLATE), ov::device::id("0")},
+};
+
+
+const std::vector<ov::AnyMap> multi_properties = {
+        {ov::device::priorities(CommonTestUtils::DEVICE_TEMPLATE), ov::enable_profiling(true)},
+        {ov::device::priorities(CommonTestUtils::DEVICE_TEMPLATE), ov::device::id("0")},
+};
+
+const std::vector<ov::AnyMap> auto_batch_properties = {
+        {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG) , CommonTestUtils::DEVICE_TEMPLATE}},
+        {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG) , CommonTestUtils::DEVICE_TEMPLATE},
+         {CONFIG_KEY(AUTO_BATCH_TIMEOUT) , "1"}},
```

Comment:
why new properties are not used here?

Reply:
@myshevts do auto batch provide new properties for this configuration values?

Reply:
per @myshevts 
let's add explicit test that AUTO:CPU supports old keys (for e.g. streams) via new API.

Reply:
> per @myshevts let's add explicit test that AUTO:CPU supports old keys (for e.g. streams) via new API.
new config key thru new API, but for the AUTO:CPU